### ISSUE TITLE
rsv_b tree.json: G_clade always "unassigned"

### ISSUE THREAD

**Issue by AngieHinrichs** [nextstrain/nextclade_data#136]
Hi & congrats on the release of nextclade v3!  

I'm using nextclade v3.0.0 and getting the datasets for RSV-A and RSV-B using these commands:
```
    nextclade dataset get --name rsv_a --output-zip rsv_a.zip
    nextclade dataset get --name rsv_b --output-zip rsv_b.zip
```
and then `nextclade run -D rsv_a.zip ...` and `... rsv_b.zip ...`.

The G_clade output TSV column for rsv_a has expected values like "GA2", "GA.3.0.0" etc., but the G_clade output TSV column for rsv_b is always "unassigned".  The tree.json from rsv_a.zip has expected G_clade values annotated on nodes, but the tree.json from rsv_b.zip has `"node_attrs": {... "G_clade": {"value": "unassigned"}` for all internal nodes as far as I can tell.  

Hopefully there's just a little typo fix that can bring back the GB's!  🙂  Thanks.

**Comment by rneher** [comment]
thanks for flagging. The GAs were also messed up (different reference with slightly different coordinates screwed up the clade definition while all eyes were on the new nomenclature...)

once we have the current troubles with our release mechanisms sorted, this should be fixed via #143

**Comment by ivan-aksamentov** [comment]
This was released in https://github.com/nextstrain/nextclade_data/releases/tag/2024-01-29--10-29-43Z and should be available to all users.

**Comment by AngieHinrichs** [comment]
Thanks @rneher and @ivan-aksamentov!

While trying it out, I ran into an issue that I'm not sure whether to file under nextclade or nextclade_data: the "rsv_a" alias is still listed by `nextclade dataset list`, but `nextclade dataset get` fails to find it.  For example, if I run
```
nextclade dataset list -n rsv_a
```
then I get output showing `(shortcuts: "rsv_a", "nextstrain/rsv/a", "nextstrain/rsv/a/hRSV-A-England-397-2017")`.  It also works for rsv_b.  

However, when I run this command:
```
nextclade dataset get --name rsv_a --output-zip rsv_a.zip
```
unlike yesterday, I now get this error output:
```
Error: 
   0: Dataset not found: 'rsv_a'.

      Did you mean:
      - nextstrain/rsv/a/EPI_ISL_412866
      - nextstrain/rsv/b/EPI_ISL_1653999
      - nextstrain/sars-cov-2/XBB
      - nextstrain/sars-cov-2/BA.2
      - nextstrain/sars-cov-2/BA.2.86
      - nextstrain/sars-cov-2/wuhan-hu-1/orfs
      - nextstrain/sars-cov-2/wuhan-hu-1/proteins
      ?

      Type `nextclade dataset list` to show available datasets.

Location:
   packages_rs/nextclade-cli/src/cli/nextclade_dataset_get.rs:80

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```

Similar for rsv_b.  I will use the full names nextstrain/rsv/a/EPI_ISL_412866 and nextstrain/rsv/b/EPI_ISL_1653999, but the aliases are more convenient.

**Comment by AngieHinrichs** [comment]
Ah, even the full name of the dataset is not working in `nextclade dataset get`:
```
# nextclade --version
nextclade 3.0.0

# nextclade dataset get -n nextstrain/rsv/a/EPI_ISL_412866 --output-zip rsv_a.zip
Error: 
   0: Dataset not found: 'nextstrain/rsv/a/EPI_ISL_412866'.

      Did you mean:
      - nextstrain/rsv/a/EPI_ISL_412866
...
```

**Comment by ivan-aksamentov** [comment]
@AngieHinrichs Thanks for reporting, Angie! I could not tell whether it's a software or data issue either :) 

Turns out it's a funny bug in the software. Fixed in  https://github.com/nextstrain/nextclade/pull/1390 and released as [3.0.1](https://github.com/nextstrain/nextclade/releases/tag/3.0.1).

I tested it like this:

```bash
for v in sars-cov-2 rsv_a rsv_b; do docker run -it --rm nextstrain/nextclade:3.0.1 bash -c "nextclade dataset get -v --name=${v} --output-zip=${v}.zip; ls ${v}.zip"; done
2024-01-29 21:12:12.451 [I] http_client.rs:101: HTTP 'GET' request to 'https://data.clades.nextstrain.org/v3/index.json'
2024-01-29 21:12:12.901 [I] http_client.rs:101: HTTP 'GET' request to 'https://data.clades.nextstrain.org/v3/nextstrain/sars-cov-2/wuhan-hu-1/orfs/2024-01-16--20-31-02Z/dataset.zip'
sars-cov-2.zip
2024-01-29 21:12:13.488 [I] http_client.rs:101: HTTP 'GET' request to 'https://data.clades.nextstrain.org/v3/index.json'
2024-01-29 21:12:13.936 [I] http_client.rs:101: HTTP 'GET' request to 'https://data.clades.nextstrain.org/v3/nextstrain/rsv/a/EPI_ISL_412866/2024-01-29--10-29-43Z/dataset.zip'
rsv_a.zip
2024-01-29 21:12:14.581 [I] http_client.rs:101: HTTP 'GET' request to 'https://data.clades.nextstrain.org/v3/index.json'
2024-01-29 21:12:15.023 [I] http_client.rs:101: HTTP 'GET' request to 'https://data.clades.nextstrain.org/v3/nextstrain/rsv/b/EPI_ISL_1653999/2024-01-29--10-29-43Z/dataset.zip'
rsv_b.zip

```


Let us know if it now works on your side!
